gobcore/model/gobmodel.json

```diff
@@ -39,6 +39,30 @@
           "json": {
             "type": "GOB.JSON",
             "description": "JSON value."
+          },
+          "datetime": {
+            "type": "GOB.DateTime",
+            "description": "DateTime value."
+          },
+          "pkinteger": {
+            "type": "GOB.PKInteger",
+            "description": "Primary key integer value"
+          },
+          "reference": {
+            "type": "GOB.Reference",
+            "description": "Reference value."
```

Comment:
A GOB.Reference needs a "ref" attribute with a reference to another 'catalog:collection'. For this test a reference can be made to a non-existing collection e.g. 'test_catalogue:test_reference'